cli/js/40_jupyter.js

```diff
@@ -92,9 +92,14 @@ function isDataFrameLike(obj) {
     return false;
   }
   const df = obj;
-  return df.schema !== void 0 && typeof df.schema === "object" &&
-    df.head !== void 0 && typeof df.head === "function" &&
-    df.toRecords !== void 0 && typeof df.toRecords === "function";
+  return (
+    df.schema !== void 0 &&
+    typeof df.schema === "object" &&
+    df.head !== void 0 &&
+    typeof df.head === "function" &&
+    df.toRecords !== void 0 &&
+    typeof df.toRecords === "function"
+  );
```

Comment:
This reformatting must have been caused by defaults in Zed. 😅

Reply:
I'll revert that before merging